secure_software_development_fundamentals.md

```diff
@@ -3898,13 +3900,13 @@ As hinted at earlier, it is critical that you use existing well-respected implem
 
 [x] Curve25519 is a widely-used algorithm that is generally considered secure.
 
-### Cryptographic Pseudo-Random Number Generator (PRNG)
+### Cryptographically Secure Pseudo-Random Number Generator (PRNG)
```

Comment:
Would be good to use one abbreviation over the whole text, so either PRNG or CSPRNG. The other patch uses CSPRNG only.

Reply:
No, because they're not the same thing. Some PRNGs are CSPRNGs, while other PRNGs are not.

Reply:
Here's what I'll do. The *title* will use CSPRNG (since that's the main topic). The text itself will explain that CSPRNGs are a subset of PRNGs, and expand both terms. I think other literature may become confusing if we don't note that not all PRNGs are CSPRNGs.

Reply:
What we have here in the title in words says CSPRNG and the abbrevation is PRNG. That is what got me confused :)

Reply:
The abbreviation was an abbreviation of *part* of the title, but I see your point. It's not longer an issue!